infrastructure/modules/serverless/data.tf

```diff
@@ -12,6 +12,3 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
Remove this file, if we don't have any code in this.

Reply:
Removed